registry/application/services/service_publisher_service.py

```diff
@@ -27,7 +30,16 @@ def save_service(self, payload):
         pass
 
     def create_service(self, payload):
-        pass
+        service_uuid = uuid4().hex
+        service = Service(
+            self.org_uuid, service_uuid, payload.get("service_id", ""), payload.get("display_name", ""),
+            payload.get("short_description", ""), payload.get("description", ""), payload.get("project_url", ""),
+            payload.get("proto", {}), payload.get("assets", {}), payload.get("ranking", DEFAULT_SERVICE_RANKING),
+            payload.get("rating", {}), payload.get("contributors", []), payload.get("metadata_ipfs_hash", ""),
+            payload.get("groups", []), ServiceStatus.DRAFT.value)
+        service_db_model = ServiceFactory().convert_entity_model_to_service_db_model(service)
+        ServiceRepository().add_service(service_db_model)
```

Comment:
Pass domain entity to repository layer Directly and use there. Domain layer should not be worry about the object that is going be made in db layer.